test/com/gu/contentapi/sanity/RunCODETests.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.contentapi.sanity
+
+import com.gu.contentapi.sanity.support.{DoNothing, FakeAppSupport}
+import play.api.test.Helpers.running
+
+/**
+ * Run all the tests that are suitable for execution against the CODE environment.
+ *
+ * Use `sbt test:run` to run this class.
+ *
+ */
+object RunCODETests extends App {
```

Comment:
You should set up a project in TeamCity to use your new command to run the `CODE` tests and check they pass.